src/javascript/ContentEditor.register.Utils.js

```diff
@@ -0,0 +1,65 @@
+export const removeMixinFromSection = (mixin, sections) => {
+    let removedFields = [];
```

Comment:
This function needs to be split in two, one to enable / disable a fieldset, the other to move a field to a new location (fieldset / field)